### DISCUSSION TITLE
2023년 08월 16일

### DISCUSSION THREAD

**Discussion by giscus** [Cyma-s/Cyma-s.github.io#75] [General]
# 2023년 08월 16일

[ 2023-08-01 ] [ 2023-08-03 ] [ 2023-08-04 ] [ 2023-08-05 ] [ 2023-08-06 ] [ 2023-08-07 ] [ 2023-08-09 ] [ 2023-08-10 ] [ 2023-08-16 ] [ 20…

https://vero.wiki/2023-08-16/

**Comment by carsago** [comment]
트랜잭션이 read only = true가 아니라면 다시 영속 상태가 되나요?

**Reply by Cyma-s** [reply]
저거 잘못 썼어요 ㅋㅋㅋㅋㅋ 읽기 전용 트랜잭션과는 관련 없습니다
https://vero.wiki/interceptor-argument-resolver-authenticated/ 이 글 참고하시죠 ^^
